joindinapp/joindinapp-Info.plist

```diff
@@ -37,13 +37,24 @@
 	<string>1.6</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
```

Comment:
I see why you added this exception, but I reckon we should move over to using https sooner rather than later (especially considering a large userbase probably use the app on public wifi).

Thoughts?